ccm_web/server/src/canvas/canvas.interfaces.ts

```diff
@@ -223,6 +223,11 @@ export interface CanvasUniqueIdErrorsBody extends CanvasErrorsBody {
   }
 }
 
+export interface CanvasUserCondensed {
+  loginId: string
+  name: string
+}
```

Comment:
Couple things here:
1) Let's move this so it's next to the other `CanvasUser` interface.
2) This interface could be based on `CanvasUser`, or vice versa. I guess I see this as a limited subset of the existing object, so we could use [`Pick`](https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys) here, and not redefine `login_id` as `loginId`.
```
type CondensedCanvasUser = Pick<CanvasUser, 'login_id' | 'name'>
```